test/pangolin/pangolin.test.js

```diff
@@ -0,0 +1,204 @@
+const { expect } = require("chai");
```

Comment:
move `test/pangolin/pangolin.test.js` to `test/avalanche/pangolin/pangolin.test.js`